wagtail/admin/utils.py

```diff
@@ -68,5 +68,5 @@ def set_query_params(url: str, params: dict):
     querydict = parse_qs(query)
     querydict.update(params)
     querydict = {key: value for key, value in querydict.items() if value is not None}
-    query = urlencode(querydict)
+    query = urlencode(querydict, doseq=True)
```

Comment:
I missed this in #11232, per https://docs.python.org/3/library/urllib.parse.html#urllib.parse.parse_qs:

> Use the [`urllib.parse.urlencode()`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode) function (with the `doseq` parameter set to `True`) to convert such dictionaries into query strings.

Otherwise the existing query params will be broken as it will ignore list values (which `parse_qs` produces) and encode it as URL-encoded `[ ... ]`. Found this while writing the tests for this PR.

Will backport this commit (only) to 5.2.x